supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -576,9 +576,16 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
     }
   }
 
-  private def handleSuccess(outputMappings: Try[JobOutputs], returnCode: Int, executionHandle: ExecutionHandle): ExecutionHandle = {
+  private def gatherJobOutputFiles: Map[String,String] = {
+      val callOutputFiles = List(jesCallPaths.stderrPath, jesCallPaths.returnCodePath, jesCallPaths.stdoutPath, gcsExecPath)
+      val mapOfFiles = callOutputFiles map { case(path) => path.getFileName.toString -> path.toString } toMap
+      val result = mapOfFiles ++ Map("callRootPath" -> callRootPath.toString)
```

Comment:
It's fine that this gets sent along, but it's essentially stuffed in a Map of detritus even though it isn't really detritus.  So I was thinking there might be a type that wraps this and the detritus map.